source/wp-content/themes/wporg-themes-2024/src/theme-downloads/view.js

```diff
@@ -0,0 +1,82 @@
+/* global google, jQuery */
+
+async function renderChart( element ) {
+	const { theme = '', labelDate, labelValue } = element.dataset;
+	const url = 'https://api.wordpress.org/stats/themes/1.0/downloads.php?slug=' + theme + '&limit=260&callback=?';
+	const downloads = await jQuery.getJSON( url );
+
+	google.charts.setOnLoadCallback( function () {
+		const data = new google.visualization.DataTable();
+		let count = 0;
+
+		data.addColumn( 'string', labelDate );
+		data.addColumn( 'number', labelValue );
+
+		Object.entries( downloads ).forEach( ( [ key, value ] ) => {
+			data.addRow();
+			data.setValue( count, 0, new Date( key ).toLocaleDateString() );
+			data.setValue( count, 1, Number( value ) );
+			count++;
+		} );
+
+		new google.visualization.ColumnChart( element ).draw( data, {
+			title: 'Density of Precious Metals, in g/cm^3',
+			colors: [ '#3858e9' ],
+			fontName: 'var(--wp--preset--font-family--inter)',
+			legend: {
+				position: 'none',
+			},
+			chartArea: {
+				height: 320,
+				bottom: 60,
+				left: 60,
+				width: '100%',
+			},
+			hAxis: {
+				textStyle: {
+					color: '#1e1e1e',
```

Comment:
We could grab this from the body if we wanted to:
```
getComputedStyle(document.body).getPropertyValue('--wp--preset--color--charcoal-1');
```

Reply:
Good idea! I've set up a colors object for all the colors I use: https://github.com/WordPress/wporg-theme-directory/pull/54/commits/a0a89821c344b95ee749335b5a66e8594c87602c